third_party/move/tools/move-package/src/compilation/model_builder.rs

```diff
@@ -135,6 +130,7 @@ impl ModelBuilder {
                 ModelBuilderOptions::default(),
                 skip_attribute_checks,
                 known_attributes,
+                warnings_are_errors, // warnings_are_errors
```

Comment:
Remove redundant (and somewhat out-of-place looking) comment?

Reply:
oops.